common/static/common/js/discussion/views/discussion_content_view.js

```diff
@@ -286,6 +286,14 @@
                             ngettext('{numVotes} Vote', '{numVotes} Votes', numVotes),
                             {numVotes: numVotes});
                         button.find('.vote-count').empty().text(votesText);
+
+                        if (this.$el.prop('className') === 'thread-content-wrapper') {
```

Comment:
`$(selector).hasClass(classname)`
Kindly use this, so if we add another class later-on, our will not break.

Reply:
Nice.

Done.